project_dev_docs/phase1_resources/interview_script.md

```diff
@@ -0,0 +1,44 @@
+# Interview Script for User Research into Open Practices
+
+**Running time:** 30-45 mins
+
+## Context
+
+_What do we want to find out?_
+
+- Interview different people across different themes and at different levels (for example, researcher, project manager, project lead, and so on)
+- Assess the current attitudes towards open working at the Turing
+- Uncover barriers and hidden fears towards open working
+- Understand what Turing people need to address these concerns, what processes need to be put in place, for example
+- Not promoting open working, rather gaining an understanding of the barriers and drivers within the Turing community
+
+## Aims
+
+1. Understand how researchers, theme leaders, project management and research engineers at the Alan Turing Institute define open practices
+2. Understand to what extent these groups are already working in line with open practices in their daily roles
+3. Understand the main barriers and drivers for adoption of working proactices across the Alan Turing Institute
+
+## Questions
+
+1. How would you define the term "open practices" and what does it mean to you in the context of your daily work? [_Answers Aim 1_]
+2. How would you define the following terms? [_Answers Aim 1_]
+   1. Open working
+   2. Open source
+   3. Open software
+   4. Open science
+3. From your perspective, what are the key similarities between the four terms? [_Answers Aim 1_]
+4. From your perspective, what are the key differences between the four terms? [_Answers Aim 1_]
+5. What do you view as the main benefits of open working?
+6. What do you view as the main drawbacks of open working?
+7. To what extent do you consider yourself working openly, in the context of the work you do at the Turing? How would you qualify that? [_Answers Aim 2_]
+   1. [_If they don't work openly_] If you don't work openly at all, can you explain in more detail the barriers to working openly in this context?
+   2. [_If you require more detail_] Can you explain in more detail how you consider yourself to be working openly?
+8. To what extent do you consider open practices to be a part of the workplace culture at the Alan Turing Instite? [_Answers Aim 2_]
+9. To what extent do you consider open practices to be **encouraged** at the Alan Turing Institute? [_Answers Aim 3_]
+   1. How are they being encouraged?
+10. [_If they have not answered the above_] What are the main barriers to adopting open practices at the Alan Turing Institute? [_Answers Aim 3_]
+11. In your opinion, what are some practical steps the Institute could take to removing or lowering the barriers to open practices? [_Answers Aim 3_]
+12. Are there any examples of open working you have experienced outside the Institute?
+    1. [_Probe_] Can you provide more detail?
+    2. [_Probe_] What do you think the Institute could learn from these examples?
+13. Are there any other comments or input you'd like to offer?
```

Comment:
Could also end with:

- "Is there anything that I haven't asked you about that you think I should have?" (allow 5-10 mins for this in case it unearths something really interesting)
- "Who else should I speak to about this?"